app/templates/application.hbs

```diff
@@ -1,13 +1,21 @@
-{{page-title "My"}}
+{{page-title "My | Real Dev Squad"}}
 
-<Navbar @firstName={{@model.firstName}} @profilePictureURL={{@model.profilePictureURL}} @signOutHandler={{this.signOutHandler}} />
+<Navbar
+  @firstName={{@model.firstName}}
+  @profilePictureURL={{@model.profilePictureURL}}
+  @signOutHandler={{this.signOutHandler}}
+/>
 
 <div class="main-container">
   {{outlet}}
 </div>
 
 <footer class="footer">Contents of this website are deployed from this
-  <a href="https://github.com/Real-Dev-Squad/website-my" target="_blank" rel="noopener noreferrer">
+  <a
+    href="https://github.com/Real-Dev-Squad/website-my'"
```

Comment:
Can we please remove the extra single inverted comma from the end?
Also, I think to avoid mistakes like these we should store this in a constant.

Reply:
Do find if this URL is already present in some constant file.

Reply:
Hey @rohan09-raj , updated the code with constant for this particular file.

Added a url.js file in constants which houses it.

Let me know if any other changes are required.